weapons.xml

```diff
@@ -0,0 +1,2309 @@
+      <Piece id="crpg_peasant_hatchet_1_t1_handle" Type="Handle" scale_factor="147" />
+    </Pieces>
+  </CraftedItem>
+  <CraftedItem id="crpg_peasant_sickle_1_t1" name="{=4wdgiTta}Sickle" crafting_template="crpg_OneHandedAxe" modifier_group="cheap_weapon">
+    <Pieces>
+      <Piece id="crpg_peasant_sickle_1_t1_blade" Type="Blade" scale_factor="100" />
+      <Piece id="crpg_peasant_sickle_1_t1_handle" Type="Handle" scale_factor="100" />
+    </Pieces>
+  </CraftedItem>
+  <CraftedItem id="crpg_peasant_hammer_1_t1" name="{=6WcZ5Tr0}Blacksmith Hammer" crafting_template="crpg_Mace" modifier_group="cheap_weapon">
+    <Pieces>
+      <Piece id="crpg_peasant_hammer_1_t1_blade" Type="Blade" scale_factor="100" />
+      <Piece id="crpg_peasant_hammer_1_t1_handle" Type="Handle" scale_factor="100" />
+    </Pieces>
+  </CraftedItem>
+  <CraftedItem id="crpg_peasant_hammer_2_t1" name="{=beVFwINO}Wooden Hammer" crafting_template="crpg_Mace" modifier_group="cheap_weapon">
+    <Pieces>
+      <Piece id="crpg_peasant_hammer_2_t1_blade" Type="Blade" />
+      <Piece id="crpg_peasant_hammer_2_t1_handle" Type="Handle" />
+      <Piece id="crpg_peasant_hammer_2_t1_pommel" Type="Pommel" />
+    </Pieces>
+  </CraftedItem>
+  <CraftedItem id="crpg_peasant_2haxe_1_t1" name="{=7phSQw5i}Hoe" crafting_template="crpg_TwoHandedAxe" modifier_group="cheap_weapon">
+    <Pieces>
+      <Piece id="crpg_peasant_2haxe_1_t1_blade" Type="Blade" scale_factor="100" />
+      <Piece id="crpg_peasant_2haxe_1_t1_handle" Type="Handle" scale_factor="100" />
+    </Pieces>
+  </CraftedItem>
+  <CraftedItem id="crpg_peasant_pitchfork_1_t1" name="{=oFEzCTpW}Iron Pitchfork" crafting_template="crpg_TwoHandedPolearm" modifier_group="cheap_weapon">
+    <Pieces>
+      <Piece id="crpg_peasant_pitchfork_1_t1_blade" Type="Blade" scale_factor="100" />
+      <Piece id="crpg_peasant_pitchfork_1_t1_handle" Type="Handle" scale_factor="100" />
+    </Pieces>
+  </CraftedItem>
+  <CraftedItem id="crpg_peasant_pitchfork_2_t1" name="{=JBVAeXx3}Pitchfork" crafting_template="crpg_TwoHandedPolearm" modifier_group="cheap_weapon">
+    <Pieces>
+      <Piece id="crpg_peasant_pitchfork_2_t1_blade" Type="Blade" scale_factor="100" />
+      <Piece id="crpg_peasant_pitchfork_2_t1_handle" Type="Handle" scale_factor="100" />
+    </Pieces>
+  </CraftedItem>
+  <CraftedItem id="crpg_peasant_polearm_1_t1" name="{=aGbTPme1}Scythe" crafting_template="crpg_TwoHandedPolearm" modifier_group="cheap_weapon">
+    <Pieces>
+      <Piece id="crpg_peasant_polearm_1_t1_blade" Type="Blade" scale_factor="103" />
+      <Piece id="crpg_peasant_polearm_1_t1_handle" Type="Handle" scale_factor="102" />
+    </Pieces>
+  </CraftedItem>
+  <Item id="crpg_training_bow" name="{=SUSbUAEl}Practice Bow" body_name="bo_composite_shotbow_b" mesh="shortbow_b" weight="0.15" difficulty="0" appearance="1" is_merchandise="false" Type="Bow" item_holsters="bow_back_2:bow_hip:bow_hip_2:bow_back">

```

Comment:
Your weapon.xml is not in the right folder...

Reply:
will fix it i always do that im a derp lol